src/test/java/org/apache/commons/compress/archivers/zip/ZipSplitOutputStreamTest.java

```diff
@@ -69,19 +69,19 @@ public void splitZipBeginsWithZipSplitSignature() throws IOException {
     public void testCreateSplittedFiles() throws IOException {
         final File testOutputFile = new File(dir, "testCreateSplittedFiles.zip");
         final int splitSize = 100 * 1024; /* 100KB */
-        final ZipSplitOutputStream zipSplitOutputStream = new ZipSplitOutputStream(testOutputFile, splitSize);
+        final ZipSplitOutputStream ZipSplitOutputStream = new ZipSplitOutputStream(testOutputFile, splitSize);
```

Comment:
-1: Local var names never start with a CAPITAL letter.

Reply:
My mistake, don't know how this happened or why I modified this test :/